ui/panels/DatabasePanel.tsx

```diff
@@ -115,6 +115,17 @@ export function DatabasePanelDetails({
           </React.Fragment>
         )}
       </div>
+      <div className="form-row">
```

Comment:
This doesn't need to be in here. It can always default on for databases since ORDER is never guaranteed in a database

Reply:
Okay